include/effectmanager.h

```diff
@@ -202,19 +198,23 @@ class EffectManager : public IJSONSerializable
         if (_vEffects.size() == 0)
             return false;
 
-        _abEffectEnabled = std::make_unique<bool[]>(_vEffects.size());
-
-        // Try to load effect enabled state from JSON also, default to "enabled" otherwise
-        JsonArrayConst enabledArray = jsonObject["eef"].as<JsonArrayConst>();
-        int enabledSize = enabledArray.isNull() ? 0 : enabledArray.size();
-
-        for (int i = 0; i < _vEffects.size(); i++)
+        if (jsonObject.containsKey("eef"))

```

Comment:
Might be worth a comment what eef ivl and so on are...

Reply:
Fair point, will add one.